docs/ARCHITECTURE.md

```diff
@@ -201,3 +201,67 @@ can be found under various directories such as `factory/`, `modprobe.d/`, `netwo
 `tools/`, `coccinelle/`, `.github/`, `.semaphore/`, `.mkosi/` host various
 utilities and scripts that are used by maintainers and developers. They are not
 shipped or installed.
+
+## UEFI
+The systemd project provides a UEFI bootloader, `systemd-boot`, and a stub that can be attached to a kernel
+image to allow loading it as a UEFI executable, `systemd-stub`. These components fully support UEFI Secure
+Boot, and this paragraph will describe their security posture and how they comply with industry-standard
+expectations for UEFI Secure Boot workflows.
+
+### Core Design
+The core design goals for these components is simplicity, achieved by offloading security-critical tasks to
```

Comment:
well, sd-stub/sd-boot are not that simple anymore... but their security model is. hence say:

"The fundamental security design goals for these components are separation of security policy logic from the rest of the functionality, achieved by...